docs/topics/wildfly-swarm/docs/concepts/packaging-types.adoc

```diff
@@ -53,5 +52,43 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Built Hollow JARs
+
+{Thorntail} ships the following pre-built hollow JARs:
+
+ifndef::product[full:: The main Java EE related capabilities]
+web:: A smaller functional scope, focused on web technologies
```

Comment:
I've _just_ noticed (sorry) that in case of product, this will read wrong. "Smaller" clearly refers to the previous line, which won't be present in product.

Reply:
Thank you for spotting this. Will fix.